docs/socialaccount/providers/saml.rst

```diff
@@ -165,3 +165,27 @@ The SAML provider has the following endpoints:
 - ``/accounts/saml/<organization_slug>/sls/``: Single Logout Service URL.
 
 - ``/accounts/saml/<organization_slug>/metadata/``: Metadata URL.
+
+Tips and Notes on SAML with Django:
+
+- Most SAML IdPs require TLS (nee SSL) to be used, making testing with
+  `runserver` challenging. Make sure to configure Django to use HTTPS.
```

Comment:
Overall remark: RST needs double backticks -- can you preview this section in an RST viewer and adjust accordingly?